nixos/modules/services/networking/pounce.nix

```
@@ -0,0 +1,224 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.pounce;
+  defaultUser = "pounce";
+  configFile = pkgs.writeText "pounce" ''
+    ${optionalString (cfg.sslCa != null) "local-ca = "+cfg.sslCa}
```

```suggestion
    ${optionalString (cfg.sslCa != null) "local-ca = ${cfg.sslCa}"}
```
nested interpolations are allowed, and usually makes things easier to read.